src/TwigExtensionFactory.php

```diff
@@ -19,36 +19,32 @@
 
 class TwigExtensionFactory
 {
-    public function __invoke(ContainerInterface $container) : TwigExtension
+    public function __invoke(ContainerInterface $container): TwigExtension
     {
-        if (! $container->has(ServerUrlHelper::class)
-            && ! $container->has(\Zend\Expressive\Helper\ServerUrlHelper::class)
-        ) {
-            throw new InvalidConfigException(sprintf(
-                'Missing required `%s` dependency.',
-                ServerUrlHelper::class
-            ));
+        $serverUrlHelper = $container->has(ServerUrlHelper::class) ? ServerUrlHelper::class : (
+        $container->has(\Zend\Expressive\Helper\ServerUrlHelper::class)
+            ? \Zend\Expressive\Helper\ServerUrlHelper::class
+            : null
+        );
```

Comment:
This bit is hard to read, I would suggest to reformat it like this and use alias:
```suggestion
        $serverUrlHelper = $container->has(ServerUrlHelper::class) 
            ? ServerUrlHelper::class
            : ($container->has(ZfServerUrlHelper::class) ? ZfServerUrlHelper::class : null);
```
 
Suggesting `Zf` instead of `Zend` because it would otherwise form `ZendServer` which is unrelated product